README.md

```diff
@@ -182,6 +182,12 @@ Name of the realm.
 
 - *Default*: 'realm.example.com'
 
+domain_realms
+-----
+Hash of domains that should be mapped to correct realm.
+
+- *Default*: {}
+
```

Comment:
Please do me a favour and support my 'enhance the internal modules' initiative by adding one or two hiera examples and what it will provide in vas.conf by that.